src/common-control/common-control-styles.html

```diff
@@ -0,0 +1,8 @@
+<!-- shared styles for all custom components -->
+<!-- Styling done the Polymer way https://youtu.be/IbOaJwqLgog?t=8m11s -->

```

Comment:
I'll go ahead and merge this, but I'd like to know if there is any benefit to putting these in a dom-module. I guess I don't like the idea of having an .html file that is really a .css file.